libs/domain/search/facet-range/facet-range.component.ts

```diff
@@ -0,0 +1,153 @@
+} from './facet-range.model';
+import { searchRangeFacetStyles } from './facet-range.styles';
+
+export class SearchRangeFacetComponent
+  extends LitElement
+  implements SearchFacetRangeComponentAttributes
+{
+  static styles = [searchRangeFacetStyles];
+
+  protected controller = new FacetController(this);
+
+  @signalProperty() name?: string;
+  @signalProperty({ type: Boolean }) open = false;
+  @signalProperty({ type: Boolean }) enableClear = true;
+  @signalProperty({ type: Number }) step = 1;
+  @signalProperty() labelMin?: string;
+  @signalProperty() labelMax?: string;
+
+  @state() min?: number;
+  @state() max?: number;
+
+  protected facet = computed(() => this.controller.getFacet() as RangeFacet);
+
+  protected $isDirty = computed(() => {
+    const {
+      values: { min, max, selected },
+    } = this.facet();
+
+    return selected?.min !== min || selected?.max !== max;
+  });
+
+  @elementEffect()
+  protected $syncValues = effect(() => {
+    const facet = this.facet();
+
+    if (!facet) return;
+
+    const {
+      values: { min, max, selected },
+    } = facet;
+
+    this.min = selected?.min ?? min;
+    this.max = selected?.max ?? max;
+  });
+
+  protected onRangeChange = debounce(
+    (e: CustomEvent<MultiRangeChangeEvent>): void => {
+      const { minValue: min, maxValue: max } = e.detail;
+      const selected = { min, max };
+
+      if (this.hasChangedValue(selected)) {
+        this.min = min;
+        this.max = max;
+        this.controller.dispatchSelectEvent({ selected });
+      }
+    },
+    300
+  );
+
+  protected onBlur(e: InputEvent): void {
+    const { value, name } = e.target as HTMLInputElement;
+    this[name as keyof SearchFacetRangeComponentValues] = Number(value);
+  }
+
+  protected onKeydown(e: KeyboardEvent): void {
+    if (e.key === 'Enter') {
+      (e.target as HTMLInputElement).blur();
+    }
+  }
+
+  protected hasChangedValue({ min, max }: RangeFacetValue): boolean {
+    const {
+      values: { selected },
+    } = this.facet();
+    return selected?.min !== min || selected?.max !== max;
+  }
+
+  protected override render(): TemplateResult | void {
+    const facet = this.facet();
+
+    if (!facet) return;
+
+    return html`<oryx-search-facet-value-navigation
+      ?open="${this.open}"
+      ?enableClear="${this.enableClear}"
+      ?dirty="${this.$isDirty()}"
+      .heading="${this.name}"
+    >
+      <section>${this.renderControls(facet)}</section>
+    </oryx-search-facet-value-navigation>`;
+  }
+
+  protected renderInput(
+    name: keyof SearchFacetRangeComponentValues,
+    min: number,
+    max: number,
+    label: string = name
+  ): TemplateResult {
+    return html`
+      <oryx-input .label=${label}>
+        <input
+          aria-label=${ifDefined(label)}
+          name=${name}
+          type="number"
+          .value=${String(this[name])}
+          min=${min}
+          max=${max}
+          step=${this.step}
+          @blur=${this.onBlur}
```

Comment:
The input is changing on blur or enter, but the slider instantly changes the inputs on drag'n drop. Wondering if this is deliberately? I can imagine that changing instantly is too heavy on the list results, but that's more because the backend might be slow. Any takes on the ux here?

Reply:
Slider is debounced by 300ms, check implementation of onRangeChange method

Reply:
In result component emit a select event, and how to handle it is out of component's scope, logic can be various

Reply:
Agreed to keep this as-is for now.